### ISSUE TITLE
Add country codes

### ISSUE THREAD

**Issue by jufy111** [imthenachoman/unraid-templates#5]
Not really an issue, more a quality of life thing as I spent far to long tracking these down.
It would be good to put this list in the readme and or/docker description

To connect to a server in a particular country 

2, Albania
10, Argentina
13, Australia
14, Austria
21, Belgium
27, Bosnia and Herzegovina
30, Brazil
33, Bulgaria
38, Canada
43, Chile
52, Costa Rica
54, Croatia
56, Cyprus
57, Czech Republic
58, Denmark
68, Estonia
73, Finland
74, France
80, Georgia
81, Germany
84, Greece
97, Hong Kong
98, Hungary
99, Iceland
100, India
101, Indonesia
104, Ireland
105, Israel
106, Italy
108, Japan
114, South Korea
119, Latvia
125, Lithuania
126, Luxembourg
128, North Macedonia
131, Malaysia
140, Mexico
142, Moldova
153, Netherlands
156, New Zealand
163, Norway
174, Poland
175, Portugal
179, Romania
192, Serbia
195, Singapore
196, Slovakia
197, Slovenia
200, South Africa
202, Spain
208, Sweden
209, Switzerland
211, Taiwan
214, Thailand
220, Turkey
225, Ukraine
227, United Kingdom
228, United States
234, Vietnam

**Comment by imthenachoman** [comment]
This is a good list. My hesitation to add this is that Nord may make changes at any time and it'll be difficult to stay in sync. Nord also has a slew of other filter types that users may want. I am going to look at the API more closely to see if I can make a better UI for it. Thanks!

**Comment by jufy111** [comment]
Yeah, it's possible that they change it but I think unlikely. The info was hard to find (even looking through their API for their server list [https://nordvpn.com/api/server](url), country codes don't appear to be listed. 

It's more just a quality of life thing, having basic instructions available in the description so that people who don't have their head fully wrapped around this stuff (like myself) don't have to go searching too far for it.

**Comment by imthenachoman** [comment]
I saw a website that had a wizard where you could specify all of the Nord VPN settings you wanted and it would spit out the config. That would be best I think. I can't find the website now though. :/
